analysis/vertfn/analyzer.go

```diff
@@ -25,34 +27,53 @@ var (
 	colorize bool
 )
 
+type RefKind string
+
+const (
+	Func     RefKind = "func"
+	Type     RefKind = "type"
+	RecvType RefKind = "recvtype"
+	Value    RefKind = "value"
+)
+
+type Direction string
+
+const (
+	Down   Direction = "down"
+	Up     Direction = "up"
+	Ignore Direction = "ignore"
```

Comment:
I guess natural zero value here fits
```suggestion
	Ignore Direction = ""
```

Reply:
With an empty value for ignore it would be harder for a user to specify ignore as a value in the configuration. I could update the flag parsing but that seems like unnecessary complexity

Reply:
it looks default value if not specified up or down, is ignore, so natural to keep it as zero value